account_move_name_sequence/__manifest__.py

```diff
@@ -12,7 +12,7 @@
     "license": "AGPL-3",
     "summary": "Generate journal entry number from sequence",
     "author": "Akretion,Vauxoo,Odoo Community Association (OCA)",
-    "maintainers": ["alexis-via", "moylop260", "frahikLV"],
```

Comment:
Don't remove the old maintainers, please

(Almost they requires explicitly remove it)

Reply:
I have @frahikLV's confirmation to remove it. In fact, that account is no longer available as it was only linked to the company email.